File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        map<int,int>mp;
        bool flag = false;
        for(int x: nums){
            if (mp[x]==0){
                mp[x]=1;
            }
            else{
                flag = true;
                break;
            }
        }
        return flag;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:

        hashmap = {}

        for x in nums:
            if x not in hashmap:
                hashmap[x] = 1
            else:
                hashmap[x] += 1

        arr = []

        for key in hashmap:
            arr.append((hashmap[key], key))

        arr.sort(reverse=True)

        res = []

        for i in range(k):
            res.append(arr[i][1])

        return res